turbinia/workers/volatility.py

```diff
@@ -60,18 +59,14 @@ def run(self, evidence, result):
             output_file_path).split()
 
     result.log('Running vol as [{0:s}]'.format(' '.join(cmd)))
-    res = self.execute(
-        cmd, result, log_files=[debug_log], new_evidence=[output_evidence],
-        close=True)
+    res = self.execute(cmd, result, new_evidence=[output_evidence], close=True)
 
     if res == 0:
-      # Write the report to the output file.
-      with open(output_file_path, 'w') as fh:
-        fh.write(output_evidence.text_data.encode('utf-8'))
+      # Get report from the output file.
+      with open(output_file_path, 'r') as fh:
+        output_evidence.text_data = fh.read().decode('utf-8')
```

Comment:
What if file_size > 16G ?

Reply:
This is just for the report text, not the memory image, so I'm not too worried (usually they are just a few kb).

Reply:
If you don't control the size of the file I opt to check, otherwise this could invoke the OOM killer

Reply:
@joachimmetz  PTAL -- I added a max size that the report will be truncated to if it is too large.  I also added some tests.